.github/workflows/api.yml

```diff
@@ -7,8 +7,6 @@ on:
       - 'packages/api/**'
       - '.github/workflows/api.yml'
   pull_request:
```

Comment:
We should be able to run CI tests for other branch targets, like here where we target the DB migration branch